setup.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+from setuptools import setup, find_packages
+
+
+REQUIRES = [
+    # core
+    "numpy",
+    "pandas==0.25.1",
+    "jupyter==1.0.0",
+    "torch==1.4.0",
+
+    # UI
+    "dash",
+
+    # deep RL
+    "rlpyt",
+
+    # misc missing libs
+    "flask_caching",
+    "Flask",
+]
+
+
+DEPENDENCY_LINKS = [
+    "git+https://github.com/astooke/rlpyt.git@f04f23d#egg=rlpyt-0.1.1.dev0",
+]
+
+
+SCRIPTS = [
+    "app.py",
+]
+
+
+setup(
+    name="airlab-retail",
+    # version="0.0.1",
```

Comment:
Left these commented out so you can choose to include more package information if you want.